tests/python/gaia-ui-tests/gaiatest/apps/system/regions/status_bar.py

```diff
@@ -7,11 +7,17 @@
 
 
 class StatusBar(Base):
+    _status_bar_locator = (By.ID, 'statusbar')
     _status_bar_time_locator = (By.ID, 'statusbar-time')
     _status_bar_maximized_wrapper = (By.ID, 'statusbar-maximized-wrapper')
     _status_bar_minimized_wrapper = (By.ID, 'statusbar-minimized-wrapper')
 
     @property
+    def is_status_bar_visible(self):
+        self.marionette.switch_to_frame()
```

Comment:
I would prefer to do all the switch in the same file. I think in this case, it's better to move this line [to here](https://github.com/mozilla-b2g/gaia/pull/27155/files#diff-7cc2ad97c4eb705e71a22a7021e79868R25). What do you think?